gc.md

```diff
@@ -72,17 +85,37 @@ everything else is garbage*
 - objects pointed to by local variables are roots (stack is scanned for roots)
 - DOM elements are roots (may be weakly referenced)
 
-## Two Generations 
+## Two Generations
 
-[watch](http://youtu.be/VhpdsjBUS3g?t=11m24s)
+[watch](https://youtu.be/VhpdsjBUS3g?t=11m24s)
 
 - object heap segmented into two parts (well kind of, [see below](#heap-organization-in-detail))
```

Comment:
Heap consists of young generation and old generation.
Young generation consists of new space.
Old generation consists of old space, map space, code space, large object space.

Reply:
OK I'll clarify that.

Reply:
Addressed in 9d79c90